packages/zipkin-instrumentation-koa/README.md

```diff
@@ -0,0 +1,20 @@
+# zipkin-instrumentation-koa
+
+Zipkin middleware for Koa 2.x that traces incoming HTTP requests.
```

Comment:
2.x or 2.7+? sometimes there are features that don't work so best to mention the minimum version known to work with the hooks even if that is the same as you are using.